src/components/ProductDetail/ProductDetail.js

```diff
@@ -42,6 +40,7 @@ class ProductDetail extends Component {
   static propTypes = {
     classes: PropTypes.object,
     product: PropTypes.object,
+    shop: PropTypes.object,
```

Comment:
Since you're only using `code` as `currencyCode` in the functions `determineProductPrice()` and `render()` why not just pass just that? And also make it required.

---
**Sidenote:**

`shop.currency.code` is quite specific for only needing one thing and`const { shop: { currency: { code: currencyCode } } } = this.props;` Will crash the app if that structure is not exactly defined. I added a ticket #118 about optional chaining.